tests/clients/test_client_github.py

```diff
@@ -0,0 +1,59 @@
+import pytest
+from httpx import HTTPStatusError
+from pytest_httpx import HTTPXMock
```

Comment:
I thought we said we'd use https://vcrpy.readthedocs.io/en/latest, see https://minbzk.github.io/ai-validation/projects/tad/adrs/0004-software-stack/